server/pom.xml

```diff
@@ -1186,7 +1198,6 @@
                 <groupId>org.apache.openjpa</groupId>
                 <artifactId>openjpa</artifactId>
                 <version>${openjpa.version}</version>
-                <scope>runtime</scope>
```

Comment:
you probably need to put this scope=runtime elsewhere, it's certainly here for a purpose